src/ImageClassifier/doodlenet.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2018 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+import * as tf from '@tensorflow/tfjs';
+import { getTopKClassesFromTensor } from '../utils/gettopkclasses';
+import DOODLENET_CLASSES from '../utils/DOODLENET_CLASSES';
+
+const DEFAULTS = {
+  DOODLENET_URL: 'https://cdn.jsdelivr.net/gh/ml5js/ml5-data-and-models@master/models/doodlenet/model.json',
+  IMAGE_SIZE_DOODLENET: 28,
+};
+
+function preProcess(img, size) {
+  let image;
+  if (!(img instanceof tf.Tensor)) {
+    if (img instanceof HTMLImageElement 
+      || img instanceof HTMLVideoElement 
+      || img instanceof HTMLCanvasElement
+      || img instanceof ImageData) {
+      image = tf.browser.fromPixels(img);
+    } else if (typeof img === 'object' && (img.elt instanceof HTMLImageElement 
+      || img.elt instanceof HTMLVideoElement 
+      || img.elt instanceof HTMLCanvasElement
+      || img.elt instanceof ImageData)) {
+      image = tf.browser.fromPixels(img.elt); // Handle p5.js image, video and canvas.
+    }
+  } else {
+    image = img;
+  }
```

Comment:
This entire block is not necessary because `img` is already a Tensor.  I explained that and I fixed it in https://github.com/ml5js/ml5-library/pull/1396 but no one is even looking!?!

Reply:
Thank you @lindapaiste for highlighting this! I'll be in touch to coordinate a meeting where we can look at the best way to integrate the work you did in the previous repo. Thank you for your patience with us!